regression/cbmc/phi-merge_uninitialized_values/dynamic.desc

```diff
@@ -5,8 +5,8 @@ activate-multi-line-match
 ^EXIT=0$
 ^SIGNAL=0$
 --
-^.*:\s+dynamic_object[0-9]+(@[0-9]+)?#0\)
-^.*\?\s+dynamic_object[0-9]+(@[0-9]+)?#0\s+:
+\([^\s]+guard[^\s]+ \? [^\s]+#0 : [^\s]+\)
+\([^\s]+guard[^\s]+ \? [^\s]+#0 : [^\s]+\)
```

Comment:
Here and below: these are duplicates, should be checking for the `else` position